lib/administrate/engine.rb

```diff
@@ -24,7 +24,13 @@ class Engine < ::Rails::Engine
     @@javascripts = []
     @@stylesheets = []
 
-    Engine.config.assets.precompile << /\.(?:svg)\z/
+    if Sprockets::VERSION.start_with?("4")
+      Dir[Engine.root.join("app/assets/images/**/*.svg")].each do |image|
+        Engine.config.assets.precompile << File.expand_path(image)
```

Comment:
I think we should have an assets initializer and whitelist the SVG assets, but this would work as well :)